installers/ansible/roles/pgo-metrics/templates/prometheus-deployment.json.j2

```diff
@@ -21,8 +21,11 @@
             },
             "spec": {
                 "securityContext": {
-{% if prometheus_supplemental_groups is defined and prometheus_supplemental_groups != '' %}
-                    "supplementalGroups": [{{ prometheus_supplemental_groups }}]
+{% if grafana_supplemental_groups is defined and grafana_supplemental_groups != '' %}
+                    "fsGroup": 26,
+                    "supplementalGroups": [{{ grafana_supplemental_groups }}]
+{% else %}}
+                    "fsGroup": 26
 {% endif %}
```

Comment:
Similar to comment in the grafana-deployment file. Also, I believe you want `prometheus_supplemental_groups` here.